src/views/blocks/rsvp.php

```diff
@@ -13,27 +13,52 @@
  *
  */
 
-$event_id = $this->get( 'post_id' );
-$tickets  = $this->get( 'tickets' );
+$event_id    = $this->get( 'post_id' );
+$tickets     = $this->get( 'tickets' );
```

Comment:
On RSVP lets use `$this->get( 'rsvp' )`, that is what we talked on that Hangout.

Reply:
waiting to merge in latest 18.19 code for this.